common/include/x86emitter/legacy_instructions.h

```diff
@@ -15,1349 +15,251 @@
 
 #pragma once
 
+#if defined(__linux__) && defined(__clang__)
+#define ATTR_DEP [[deprecated]]
+#define ATTR_DEP [[deprecated]]
```

Comment:
Because I want to support GCC 4.8 and I didn't want to bother with complex ifdef. Beside, there are still hundreds of warnings. I don't want to flood important warning with noise.